about.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Green Earth Summit</title>
+</head>
+<body>
+    <nav>
+        <div class="upper-nav">
+            <ul class="top">
+                <li class="fb"><img  id="fb" src="assets/fb (1).png" alt="facebook icon"/></li>
+                <li class="twitter"><img id="twitter" src="assets/twitter.png" alt="twitter icon"></li>
+                <li class="pa"><p>English</p></li>
+                <li class="pa"><p>MyPage</p></li>
+                <li class="pa"><p>Logout</p></li>
+            </ul>
+        </div>
```

Comment:
- [ ] Kindly note that your navigation bars have to be aligned together as in the same design, kindly fix that.

| Your design | Actual design |
| ------------- | ------------- |
| ![image](https://github.com/fatmahussein/Green-Earth-Summit/assets/102612396/f828d51f-6f7e-455a-8a6c-0f634ed198fb)  | ![image](https://github.com/fatmahussein/Green-Earth-Summit/assets/102612396/031f0a31-f245-41b9-b0b1-6d5e9bfb9f48)  |